src/ast/ast_validations.c

```diff
@@ -12,6 +12,7 @@
 #include "../arithmetic/repository.h"
 #include "../arithmetic/arithmetic_expression.h"
 #include <assert.h>
+#include "../util/rax_extensions.h"
```

Comment:
```
#include "../util/rax_extensions.h"
#include <assert.h>
```

Reply:
rax_extensions.h is not part of this PR.

Reply:
i need it for rax subset

Reply:
my bad